cdap-examples/HelloWorld/src/test/java/co/cask/cdap/examples/helloworld/HelloWorldTest.java

```diff
@@ -44,6 +45,10 @@
 
   @Test
   public void test() throws Exception {
+    NamespaceId streamSpace = new NamespaceId("streamSpace");
```

Comment:
Oh my bad. I should not commit the changes to HelloWorld examples. Will remove this.

The unit test for this are here: https://github.com/caskdata/cdap/pull/6003/files#diff-47264ee491fdca6b86f1aaa624cb05b1R48. Basically I create a stream in a different namespace and deploy an app in namespace DEFAULT to test if it can connect to that stream.